- floor: 8
  altitude: 29.6
  map_name: '/eng2/8f'
- floor: 7
  altitude: 25.8
  map_name: '/eng2/7f'
  floor_name: 'eng2/7f'
  door_frame_id: '/eng2/7f/elevator_door'
  door_dimensions: [1.0, 0.5, 1.0]
  outside_pose:
    frame_id: 'map'
    position: [2.631, -30.344, -0.000]
    orientation: [0.000, 0.000, -0.861, 0.509]
  inside_pose:
    frame_id: 'map'
    position: [1.634, -33.990, -0.000]
    orientation: [-0.000, -0.000, 0.476, 0.879]
  buttons:
    down: '/eng2/7f/elevator_bot_down'
- floor: 6
  altitude: 22.0
  map_name: '/eng2/6f'
- floor: 5
  altitude: 16.8
  map_name: '/eng2/5f'
- floor: 4
  altitude: 13.1
  map_name: '/eng2/4f'
- floor: 3
  altitude: 9.1
  map_name: '/eng2/3f'
  floor_name: 'eng2/3f'
  door_frame_id: '/eng2/3f/elevator_door'
  door_dimensions: [1.0, 0.5, 1.0]
  outside_pose:
    frame_id: 'map'
    position: [2.631, -30.344, -0.000]
    orientation: [0.000, 0.000, -0.861, 0.509]
  inside_pose:
    frame_id: 'map'
    position: [1.634, -33.990, -0.000]
    orientation: [-0.000, -0.000, 0.476, 0.879]
  buttons:
    up: '/eng2/3f/elevator_bot_up'
- floor: 2
  altitude: 5.0
  map_name: '/eng2/2f'
- floor: 1
  altitude: 0
  map_name: '/eng2/1f'
  floor_name: 'eng2/1f'
  door_frame_id: '/eng2/1f/elevator_door'
  door_dimensions: [1.0, 0.5, 1.0]
  outside_pose:
    frame_id: 'map'
    position: [2.631, -30.344, -0.000]
    orientation: [0.000, 0.000, -0.861, 0.509]
  inside_pose:
    frame_id: 'map'
    position: [1.634, -33.990, -0.000]
    orientation: [-0.000, -0.000, 0.476, 0.879]
  buttons:
    up: '/eng2/1f/elevator_bot_up'
- floor: -1
  altitude: -4
  map_name: '/eng2/1f'
